docs/src/api/out.md

```diff
@@ -0,0 +1,289 @@
+# API Reference
+
+## Packages
+- [objectstorage.k8s.io/v1alpha1](#objectstoragek8siov1alpha1)
+
+
+## objectstorage.k8s.io/v1alpha1
+
+
+
+
+#### AuthenticationType
+
+_Underlying type:_ _string_
+
+
+
+
+
+_Appears in:_
+- [BucketAccessClass](#bucketaccessclass)
+
+| Field | Description |
+| --- | --- |
+| `Key` |  |
+| `IAM` |  |
+
```

Comment:
I see lots of blank space in this doc. Is this auto-generated from the API?

Reply:
Yes, this is generated using [elastic/crd-ref-docs](https://github.com/elastic/crd-ref-docs). It’s the easiest tool to use, and I think we don’t need anything more complicated.

Reply:
Got it. I won't stress over the seemingly-excessive amount of whitespace then